app/helpers/application_helper.rb

```diff
@@ -187,6 +187,14 @@ def courses
     ]
   end
 
+  def custom_button
```

Comment:
I had trouble coming up with a name and this was the best I could think of, what kind of name do you think it should have?